src/megaclient.cpp

```diff
@@ -6152,6 +6157,37 @@ void MegaClient::sc_la()
     }
 }
 
+void MegaClient::sc_ub()
+{
+    int status = 0;
+    for (;;)
+    {
+        switch (jsonsc.getnameid())
+        {
+            case 's':
+                status = (int)jsonsc.getint();
+                break;
+
+            case EOO:
+                if (status == 0)
```

Comment:
In this case, we should skip the notification, IMO. The `0` value is not even expected by the apps.

Reply:
Actually, if the master account goes back to a non-business plan, the status could be `0` (not yet implemented, but API is working on it). So perhaps allow (and add to doc) that status and use a different invalid value here. Also, remember to update the `MegaClient::businessStatus` accordingly.